spec/migrations/20200409175736_add_failed_login_attempts_to_users_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_migration
+
+describe AddFailedLoginAttemptsToUsers do
+  let(:user_stub) { migration_stub(:User) }
+  let(:user) { user_stub.create(:name => 'test', :userid => 'test') }
+
+  migration_context :up do
+    it 'creates a new field which defaults to 0' do
+      expect { user.failed_login_attempts }.to raise_error
+
+      migrate
+      user.reload
+
+      expect(user.failed_login_attempts).to eq(0)
+    end
+  end
+
+  migration_context :down do
+    it 'removes the field' do
```

Comment:
No need to test the down migration as it is only testing a built-in Rails method.